README.md

```diff
@@ -1,78 +1,66 @@
 # Apify Keboola Extractor
 
-Apify extractor for Keboola Connection
+You can use it to connect data from [Apify platform](https://apify.com/) into [Keboola platform](https://www.keboola.com/).
 
-## Test
+## Resources
 
-### Sample configuration
+* [Apify documentation](https://docs.apify.com/tutorials/integrations#get-started)
+* [Getting started tutorial](https://help.apify.com/en/articles/2003234-keboola-integration)
```

Comment:
Maybe we can do without the first link here? It just leads to the link that points to the second article anyway

Also, could you update the link texts, pls:
- Getting started tutorial => Apify/Keboola integration tutorial
- Apify component => Apify component on Keboola
- Keboola API blueprint => Keboole API reference